cmd/webhook.go

```diff
@@ -370,16 +386,22 @@ func (whsvr *WebhookServer) mutate(ar *admissionv1.AdmissionReview) *admissionv1
 	infoLogger.Printf("AdmissionReview for Kind=%v, Namespace=%v Name=%v (%v) UID=%v patchOperation=%v UserInfo=%v",
 		req.Kind, req.Namespace, req.Name, pod.Name, req.UID, req.Operation, req.UserInfo)
 
+	// Retrieves the configmap containing the list of shares. Must loop through this
+	// Format is "filer1": '["share1", "share2"]'
+	svmShareList, errorSvm := clientset.CoreV1().ConfigMaps(pod.Namespace).Get(context.Background(), existingShares, metav1.GetOptions{})
+	if k8serrors.IsNotFound(errorSvm) {
+		cmExists = false
+	}
 	// determine whether to perform mutation
-	if !mutationRequired(&pod.ObjectMeta) {
+	if !mutationRequired(&pod.ObjectMeta, cmExists) {
```

Comment:
Can't this "cmExists" logic be taken out of this function? Like, cant you just do
```
if k8serrors.IsNotFound(errorSvm) || !mutationRequired(&pod.ObjectMeta) {
    blabla
}
```